### ISSUE TITLE
Can't install pulumi_aws on Windows

### ISSUE THREAD

**Issue by maver1ck** [pulumi/pulumi-aws#587]
I'm getting following error when trying to install pulumi_aws package on windows.
```
C:\Users\maverick>pip install pulumi_aws
Collecting pulumi_aws
  Using cached https://files.pythonhosted.org/packages/b4/06/2e7604133b101b0760dbe2f2ba43de78a343987f5fff489614c86129312a/pulumi_aws-0.18.7.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\maverick\AppData\Local\Temp\pip-install-v77z2r_m\pulumi-aws\setup.py", line 20, in <module>
        long_description=readme(),
      File "C:\Users\maverick\AppData\Local\Temp\pip-install-v77z2r_m\pulumi-aws\setup.py", line 15, in readme
        return f.read()
      File "c:\users\maverick\scoop\apps\python\current\lib\encodings\cp1250.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 1099: character maps to <undefined>
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\Users\maverick\AppData\Local\Temp\pip-install-v77z2r_m\pulumi-aws\```

**Comment by lukehoban** [comment]
Could you share what python version you are using?

**Comment by maver1ck** [comment]
Python 3.7.3

**Comment by ellismg** [comment]
I suspect the problem here is that in our `setup.py` we are not passing `utf-8` as the encoding to uuse when `open`ing the `README.rst` file, which means Python will determine what encoding to use based on user preferences.  In this case, it appears that we end up using `CP-1250` (not surprising, given that windows defaults to these sorts of encodings instead of utf-8) and then things go off the rails when we try to decode some of the UTF-8 encoded characters in the `README.rst`.

We should be able to address this by changing our setup.py, which are generated by `tfgen`.  I'll open a PR for that.

**Comment by joeduffy** [comment]
This also happens on Linux. Bumping priority.

**Comment by lukehoban** [comment]
In a fresh `docker un -it ubuntu bash` I get a similar, but not identical, error:

```
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-igtf82mu/pulumi-aws/setup.py", line 20, in <module>
        long_description=readme(),
      File "/tmp/pip-build-igtf82mu/pulumi-aws/setup.py", line 15, in readme
        return f.read()
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1097: ordinal not in range(128)
```

**Comment by ellismg** [comment]
This should be fixed with the 0.18.11 release of `pulumi_aws`.  I verified that it fixed @lukehoban's case above.  Note that the install continues to fail if you don't have `pulumi` on your path, because the setup scripts try to run `pulumi plugin install` unconditionally.  I opened https://github.com/pulumi/pulumi-terraform/issues/392 to track that.
